### ISSUE TITLE
Typescript compiler error caused by FetchMock

### ISSUE THREAD

**Issue by haggs** [jefflau/jest-fetch-mock#108]
I'm on jest-fetch-mock 2.1.0, running into this compiler error in my typescript project, when I try to build.
```
Type error: Generic type 'MockInstance<T, Y>' requires 2 type argument(s).  TS2314

    17 | }
    18 | 
  > 19 | export interface FetchMock extends jest.MockInstance<any> {
       |                                    ^
    20 |     (input?: string | Request, init?: RequestInit): Promise<Response>;
    21 | 
    22 |     mockResponse(body: BodyOrFunction, init?: MockParams): FetchMock;
```

I believe line 19 should be:
```
export interface FetchMock extends jest.MockInstance<any, any> {
```

**Comment by haggs** [comment]
Ah wait, I didn't realize MockInstance was from jest. Probably something to do with my version of jest in my project...

**Comment by haggs** [comment]
I decided to leave this open because it will be an issue if you have an updated version of @types/jest. I experienced this error with @types/jest@2.3.14 and it looks like jest-fetch-mock is using 2.3.10.

**Comment by kevcenteno** [comment]
Fixed in #106
